libr/asm/p/asm_z80.c

```diff
@@ -5,7 +5,7 @@
 #include <r_lib.h>
 #include <r_asm.h>
 
-#define GPL_COMPLIANT 0
+#define GPL_COMPLIANT 1
```

Comment:
maybe we should get rid of GPL_COMPLIANT completely, just remove the case in 0 and all that cpp work